CIP-????/README.md

```diff
@@ -0,0 +1,90 @@
+---
+CIP: ???
+Title: multi-arity uplc lambda node
```

Comment:
```suggestion
Title: Multi-arity UPLC lambda node
```
All CIP titles, like all titles, are capitalised.  (Since these titles can also be considered phrases or sentences, it's not necessary to capitalise every word... some authors do & some don't.)